test/e2e/scheduling.go

```diff
@@ -202,10 +230,21 @@ func createTestObjs(tl common.TestLogger, client genericclient.Client, typeConfi
 	if !ok {
 		return "", errors.Errorf("Unable to find fixture for %q", typeConfigName)
 	}
-	fedObject, err := common.NewTestObject(typeConfig, namespace, []string{}, fixture)
+
+	clusters := []string{}
+	fedObject, err := common.NewTestObject(typeConfig, namespace, clusters, fixture)
 	if err != nil {
 		return "", err
 	}
+
+	if rspSpec.PlacementIntersection {
+		clusters = append(clusters, fedClusters[0])
+		err = util.SetClusterNames(fedObject, clusters)
```

Comment:
I think this needs to be tested with a clusterselector rather then a cluster name.